lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -51,10 +51,10 @@ describe('BasicLtiLaunchApp', () => {
       },
     });
 
-    // fake js-hypothesis-config
+    // fake js-lms-config
     fakeHypothesisConfig = sinon.stub(document, 'querySelector');
     fakeHypothesisConfig
-      .withArgs('.js-hypothesis-config')
+      .withArgs('.js-lms-config')
```

Comment:
One day maybe just 'lms-config'? Although, I do love a bit of titular redundancy

Reply:
Yep. Next thing to do is to rename this to `js-config` to match with the name of the `LTILaunchResource.js_config` property. Note that the `js-` prefix is a pattern used to denote CSS classes that exist for the purpose of enabling JavaScript to find things rather than for styling (see <https://github.com/hypothesis/frontend-toolkit/blob/master/docs/css-style-guide.md#classes-used-in-code>) so that won't be changing